dist/s_longlines

```diff
@@ -20,7 +20,7 @@ l=`(cd .. &&
 
 for f in $l ; do
 	expand -t8 < ../$f | awk -- \
-	    "{if(length(\$0) > 80) printf(\"%s:%d\\n\", \"$f\", NR)}"
+	    "{if(length(\$0) > 100) printf(\"%s:%d\\n\", \"$f\", NR)}"
```

Comment:
Updated the length of the long line because `s_all` was complaining about `src/reconcile/rec_visibility.c` file.